python/nav/ipdevpoll/config.py

```diff
@@ -72,6 +72,7 @@ class IpdevpollConfig(NAVConfigParser):
 RITTAL_WERK_RUDOLF_LOH_GMBH_COKG = RITTAL-CMC-III-MIB
 JUNIPER_NETWORKS_INC = ENTITY-SENSOR-MIB JUNIPER-DOM-MIB
 SUPERIOR_POWER_SOLUTIONS_HK_COLTD = Pwt3PhaseV1Mib
+ALCATEL_LUCENT_ENTERPRISE_FORMERLY_ALCATEL = ALCATEL-IND1-PORT-MIB
```

Comment:
Please also add this line as a commented out line in the example version of `ipdevpoll.conf` (in `python/nav/etc/`) so the default value can be observed by end users editing the config.